Added 2 new tutorials (#69)
mysteriumnetwork/documentation: content/developers/dvpntutorial.md

- **Supervisor** - a service which allows our application to interact with `node` without the need for privilege escalation. Supervisor used only for client applications.
- **Wireguard** - VPN tunnel

**Typical application startup:**

1. Start the Node.
2. Install & Start Supervisor service(if it has not been installed yet).
3. Setup connection to the Node Tequila API through the localhost.
4. Setup connection to the Supervisor through the socket.
5. Do some application logic...

Each Mysterium client Node is running Tequila API on `localhost:44050` which is used for communication and control. To interact with this API we will use the package `mysterium-vpn-js`. You can see docs for API by opening [localhost:44050/docs](http://localhost:44050/docs) in your browser(when Node working) and by visiting [mysterium-vpn-js client](https://github.com/mysteriumnetwork/mysterium-vpn-js/blob/master/src/tequilapi-client.ts)
## Let's build some simple Electron App

Clone our demo repository to your local machine

1. `git clone git@github.com:mysteriumnetwork/tequilapi-webapp-tutorial.git`
2. `yarn install`
3. `yarn dev` - to see how it looks like

### Structure & main modules

> Usually, people are building react app with the use of `create-react-app` tool which separates Source code from Electron.

Reviewer: Write in a better way

Reviewer: ```suggestion
Usually, people build react apps using the `create-react-app` tool which separates Source code from Electron.
```

**Every client application consist of these four parts:**

- **Application** - your client application
- **Mysterium Node** - client node to interact with Mysterium Network
- **Supervisor** - service used to move out from Node any services that need escalation of privileges and also used to control them. Supervisor used only for client applications.
- **Wireguard** - VPN tunnel

**Typical application startup:**

1. Start the Node.
2. Install & Start Supervisor service(if it has not been installed yet).
3. Setup connection to the Node Tequila API through the localhost.
4. Setup connection to the Supervisor through the socket.
5. Do some application logic...

Each Mysterium client Node is running Tequila API on `localhost:44050` which is used for communication and control. To interact with this API we will use the package `mysterium-vpn-js`. You can see docs for API by opening [localhost:44050/docs](http://localhost:44050/docs) in your browser(when Node working) and by visiting [mysterium-vpn-js client](https://github.com/mysteriumnetwork/mysterium-vpn-js/blob/master/src/tequilapi-client.ts)
## Let's build some simple Electron App

> Clone our demo repository to your local machine

Reviewer 2: Didn't we already have something similar at the top?
